components/eamxx/cime_config/namelist_defaults_eamxx.xml

```diff
@@ -589,18 +600,32 @@ be lost if SCREAM_HACK_XML is not enabled.
   <!-- Grids manager specs -->
   <grids_manager>
     <Type>Homme</Type>
-    <physics_grid_type>GLL</physics_grid_type>
-    <physics_grid_type hgrid=".*pg2">PG2</physics_grid_type>
-    <physics_grid_rebalance>None</physics_grid_rebalance>
-    <dynamics_namelist_file_name>./data/namelist.nl</dynamics_namelist_file_name>
-    <vertical_coordinate_filename type="file">UNSET</vertical_coordinate_filename>
-    <vertical_coordinate_filename nlev="72">${DIN_LOC_ROOT}/atm/scream/init/vertical_coordinates_L72_20220927.nc</vertical_coordinate_filename>
-    <vertical_coordinate_filename nlev="128">${DIN_LOC_ROOT}/atm/scream/init/vertical_coordinates_L128_20220927.nc</vertical_coordinate_filename>
+    <Type COMPSET=".*EACE">ACE</Type>
```

Comment:
Note: this PR will conflict with #7197. Some care will be needed when resolving conflicts to avoid undoing the de-capitalization work. Additionally, the `Homme`/`ACE` keys will have to be decapitalized too.